.github/workflows/frontend-ci-blacksmith.yml

```
@@ -0,0 +1,59 @@
+
+      - name: Run format check
+        run: yarn format:check
+
+      - name: Build application
+        run: yarn build
+        env:
+          NITRO_PRESET: node-server
+
+      - name: Run tests
+        run: yarn test
+        if: ${{ false }} # テストが設定されたら有効化
```

actionlint (1.7.10)

[error] constant expression "false" in condition. remove the if: section

(if-cond)